lambda_code/tests/test_fhir_repository.py

```diff
@@ -1,21 +1,17 @@
 import json
+import os
+import sys
 import time
 import unittest
 from unittest.mock import MagicMock, patch, ANY
 
 import botocore.exceptions
 from boto3.dynamodb.conditions import Attr, Key
 
-from fhir_repository import ImmunizationRepository, create_table
-from models.errors import ResourceNotFoundError, UnhandledResponseError
-
+sys.path.append(f"{os.path.dirname(os.path.abspath(__file__))}/../src")
```

Comment:
This should not be necessary. If your editor can't find the paths let me know and I'll have a look.